torchvision/transforms/v2/_augment.py

```diff
@@ -12,7 +12,38 @@
 from .utils import is_simple_tensor, query_chw
 
 
+# TODO: Just move that to _misc.py?
```

Comment:
Previously, `CutMix` and `MixUp` where also here. So we'll have to move all three. IMO `_augment` is fine albeit a little generic.